mcts/certsuite/cert.py

```diff
@@ -615,7 +621,7 @@ def get_application_ini(dm):
 def get_buildprop(dm):
     # get build properties
     buildprops = {}
-    buildpropoutput = dm.shellCheckOutput(["cat", "/system/build.prop"])
+    buildpropoutput = dm.shell_output("cat /system/build.prop")
```

Comment:
I understand that this is a different method, but was just wondering the details on this one. I think it should be fine either way. thanks.